test/test_other.py

```diff
@@ -9771,6 +9771,18 @@ def test_separate_dwarf_with_filename_and_path(self):
                       '-sSEPARATE_DWARF_URL=http://somewhere.com/hosted.wasm'])
     self.assertIn(b'somewhere.com/hosted.wasm', read_binary('a.out.wasm'))
 
+  def test_dwarf_system_lib(self):
```

Comment:
Can you mark this new test a `@crossplatform` so it runs on mac and windows too?

Reply:
It looks the test is failing on Windows due to path separators (could be related to WebAssembly/binaryen#2781)

@sbc100
Should we modify the test to make it pass on Windows? or drop `@crossplatform`?

Reply:
I think it would be good to fix if possible.    Perhaps if this is a pre-existing issue maybe we don't have to fix it now, but it would be good to fix if possible.

Any idea how `system\\\\lib/emmalloc.c` could end up like that?

Reply:
After some digging on my local Windows machine, It seems backslashes in `system\\\\lib/emmalloc.c` are coming from the source file path we're passing to Clang, and the slash before basename is due to [`-ffile-reproducible`](https://reviews.llvm.org/D122766).

Currently, embuilder is passing the source file path in the platform-specific format (with backslashes as separators on Windows):

`clang.exe <redacted> -c ..\..\..\system\lib\emmalloc.c`

and I get the following DWARF entry

`DW_AT_name	("system\\lib/emmalloc.c")`

Then when I replace backslashes in the source file path with slashes, the DWARF also have slashes in its file names.

`clang.exe <redacted> -c ../../../system/lib/emmalloc.c`
`DW_AT_name	("system/lib/emmalloc.c")`

Therefore, I think the best approach is to apply `utils.normalize_path` to source file names in the build command.
What do you think?

Reply:
That looks right to me.